appveyor.yml

```diff
@@ -23,3 +25,4 @@ on_finish:
             (ls $zipFile)
         ) | % { Push-AppveyorArtifact $_.FullName }
 
+os: Unstable
```

Comment:
Please end a file with a newline [See item 5 in style guidelines](https://github.com/PowerShell/DscResources/blob/master/StyleGuidelines.md)